sdks/python/container/Dockerfile

```diff
@@ -67,13 +64,33 @@ RUN ln -s /usr/bin/ccache /usr/local/bin/gcc
 RUN ccache --set-config=sloppiness=file_macro && ccache --set-config=hash_dir=false
 
 ####
-# Install Apache Beam SDK
+# Install Apache Beam SDK. Use --no-deps and pip check to verify that all
+# necessary dependencies are specified in base_image_requiremetns.txt.
 ####
 COPY target/apache-beam.tar.gz /opt/apache/beam/tars/
-RUN pip install -v /opt/apache/beam/tars/apache-beam.tar.gz[gcp]
+RUN pip install --no-deps -v /opt/apache/beam/tars/apache-beam.tar.gz[gcp]
+RUN pip check
```

Comment:
So if this fails, will the whole docker install fail?

Reply:
yes,  and that's intentional. we used to release containers with mismatched dependencies. this checked fixed it.